src/profiles.jl

```diff
@@ -137,3 +137,130 @@ function profile_solvers(
   end
   p
 end
+
+"""
+    get_profile_solvers_data(stats, costs; kwargs)
+
+Exports performance profiles plot data comparing `solvers` based on the data in `stats` in a .csv file.
```

Comment:
I don't think this is accurate as no CSV file is generated here.

Reply:
Yes, that was a mistake. The doc has been updated.